app/helpers/post_helper.rb

```diff
@@ -1,17 +1,21 @@
 module PostHelper
   def recursive_list_tag(comment)
-    html = content_tag(:ul) {
-      ul_contents = ""
-      if comment.body.nil?
-        ul_contents << link_to(content_tag(:li, "#{comment.data["count"]} More Comments"), "#").html_safe
-      else
-        ul_contents << content_tag(:li, comment.body)
-      end
-      comment.replies.each do |reply|
-        ul_contents << recursive_list_tag(reply)
-      end
+      html = content_tag(:ul) {
```

Comment:
This is great! A lot going on here. Will think about other options, but you're totally making it work, and I think a helper is the right place for this kind of logic.